tests/extension.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import time
+from deployment import DeploymentTest
+from basic import BasicDeploymentSpec
+
+EXTERNAL = {
+    "xenial": "cs:~juju-solutions/cwr-11",
+}
+
+
+class ExtensionDeploymentSpec(BasicDeploymentSpec):
+
+    def _pre_setup_00_extension(self):
+        self.deployment.add("cwr", EXTERNAL[self.series])
+        self.deployment.relate(
+            "jenkins:extension", "cwr:jenkins")
+        self.deployment.expose("cwr")
```

Comment:
The `ExtensionDeploymentTest` connects to it (on port 5000) to confirm that it can interact properly with Jenkins.